docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md

```diff
@@ -80,53 +75,39 @@ In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to c
+1. Go to **Storage accounts** and search for **sumobrlogs**, then select **sumobrlogs\<*random-string*\\>**. <br/><img src={useBaseUrl('/img/send-data/storage-accounts.png')} alt="storage-accounts" width="800"/>
+1. Under **Data Storage** do the following:
+    1. Click **Tables**.
+    1. Click **+ Table**.
+    1. Enter **FileOffsetMap** as table name and click **OK**.
 
-   1. Click **Tables**.
-   1. Click **+ Table**.
-   1. For Name, enter **FileOffsetMap**.
+<img src={useBaseUrl('/img/send-data/Azure_Blob_create-table.png')} alt="Azure_Blob_create-table" width="900"/>
 
-1. Click **OK**.
+## Example: Push NSG flow logs from a Network Security Group to Azure Blob Storage
 
-    ![Azure_Blob_create-table.png](/img/send-data/Azure_Blob_create-table.png)
-
-## Step 4. Push logs from Azure Service to Azure Blob Storage
-
-This section describes how to push logs from an Azure service to Azure Blob Storage by configuring Diagnostic Logs. The instructions use the Azure Web Apps Service as an example. 
+This section describes how to push logs from a network security group into Azure Blob Storage by configuring nsg flow Logs. The instructions use a network security group as an example. 
 
 1. Login to the Azure Portal.
-1. Click **AppServices > Your Function App > Diagnostic Logs** under **Monitoring**.
-1. You will see the Diagnostic Logs blade. Enable Application Logging, Web Server Logging, or both, and click **Storage Settings**.
-1. Select the Storage Account whose connection string you configured in [Step 1](#step-1-configure-azure-storage-account). 
-1. In the Containers blade, select the container you created in [Step 1](#step-1-configure-azure-storage-account).
-1. In the Diagnostic Logs blade, specify the **Retention Period (Days)**, and click **Save** to exit Diagnostic Logs configuration.
-
-    ![export-webapp-logs.png](/img/send-data/export-webapp-logs.png)
-
+1. Click **Network security groups > Select a network security group**.
+1. Click on **NSG flow logs** when you see it under **Monitoring**, and click **Create**.
+1. Click on **Select resource** and choose a NSG that is present in the same region as the storage account configured in [Step 1](#step-1-configure-azure-storage-account).
+1. Under **Subscription > Storage Accounts**, select the storage account configured in [Step 1](#step-1-configure-azure-storage-account).
+1. Specify the **Retention (days)** and click **Review + create**. <br/><img src={useBaseUrl('/img/send-data/review+create.png')} alt="review+create" width="700"/>
+1. Review the configuration of the flow log and click **Create**. <br/><img src={useBaseUrl('/img/send-data/review-configuration.png')} alt="review-configuration" width="600"/>
 :::tip
 If logs from Azure Blob Storage do not start to flow into Sumo, see [Troubleshoot Azure Blob Storage Log Collection](troubleshoot-azure-blob-storage-log-collection.md).
```

Comment:
```suggestion
If logs from Azure Blob Storage, do not start to flow into Sumo, see [Troubleshoot Azure Blob Storage Log Collection](troubleshoot-azure-blob-storage-log-collection.md).
```

Reply:
there shouldn't be a comma here

Reply:
A comma is required as this is an If clause statement.

Reply:
The condition for the if statement is 'logs from Azure Blob Storage do not start to flow into Sumo Logic', so we add a comma after this. And not after Storage. That's what I meant.

Reply:
@JV0812 Please add the comma here.

Reply:
@JKashyap96 You are right, this is a condition. If "logs from Azure Blob Storage do not start to flow into Sumo". We just need to write Sumo logic.